sda-commons-app-example/openapi.yaml

```diff
@@ -2,6 +2,9 @@ openapi: 3.0.1
 info:
   title: OpenAPI definition
   version: v0
+servers:
+- url: http://localhost:52960/api
+  description: Generated server url
```

Comment:
How did that end up here? 🤔

Reply:
Maybe updated by a test?

Reply:
I created a subtask. We can investigate that later.